health/micro-ui/web/packages/modules/campaign-manager/src/pages/employee/NewCampaignCreate/CampaignHome.js

```
@@ -131,51 +137,108 @@ const CampaignHome = () => {
+            <div className="digit-campaign-home-text-description">
+              {t("HCM_START_WITH_CAMPAIGN_TEMPLATE_DES")}
+            </div>
+          </div>
+          <div className="digit-campaign-home-card-count">
+            <MobileLayout width="20" height="20" fill={"#0B4B66"} />
+            <div className="digit-campaign-home-card-count-text">{`${templatesDataLength} ${t("TEMPLATES_AVAILABLE")}`}</div>
+          </div>
+        </Card>
+        <Card
+          className="digit-campaign-home-card"
+          tabIndex={1}
```

Biome (2.1.2)

[error] Avoid positive values for the tabIndex prop.

Elements with a positive tabIndex override natural page content order. This causes elements without a positive tab index to come last when navigating using a keyboard.
Use only 0 and -1 as tabIndex values. Avoid using tabIndex values greater than 0 and CSS properties that can change the order of focusable HTML elements.
Unsafe fix: Replace the tableIndex prop value with 0.


(lint/a11y/noPositiveTabindex)